src/fidesctl/core/config/credentials_settings.py

```diff
@@ -81,3 +87,18 @@ def get_config_aws_credentials(
     credentials_dict = credentials_config.get(credentials_id, None)
     parsed_config = AWSConfig.parse_obj(credentials_dict) if credentials_dict else None
     return parsed_config
+
+
+def get_config_bigquery_credentials(
+    dataset: str, credentials_config: Dict[str, Dict], credentials_id: str
+) -> Optional[BigQueryConfig]:
+    credentials_dict = credentials_config.get(credentials_id, None)
```

Comment:
```suggestion
    credentials_dict = credentials_config.get(credentials_id)
```

Just a quick fact, `None` is the default return value for `get`:

```
In [5]: foo = {"some": "value"}

In [6]: foo.get("test") is None
Out[6]: True
```